src/browser/api/system.js

```diff
@@ -610,6 +610,38 @@ exports.System = {
             }
         });
     },
+    getCookies: function(opts, callback, errorCallback) {
+        const { url, name } = opts;
+        if (url && url.length > 0 && name && name.length > 0) {
+            session.defaultSession.cookies.get({ url, name }, (error, cookies) => {
+                if (error) {
+                    log.writeToLog(1, error, true);
+                    errorCallback(`Error getting cookie ${name}`);
+                } else if (cookies.length > 0) {
+                    const data = [];
+                    for (let cookie of cookies) {
```

Comment:
:neckbeard: you could do something like `cookies.filter().map()` instead of the for loop (just a suggestion).

Reply:
Updated as requested

Reply:
Updated as requested